android/build.gradle

```diff
@@ -5,11 +5,12 @@ buildscript {
     repositories {
         google()
         jcenter()
```

Comment:
```suggestion
        mavenCentral()
```

Reply:
Always mavenCentral before jcenter, due to fraud cases.

Nice article
https://blog.autsoft.hu/a-confusing-dependency/

Reply:
I didn't know that! Thanks for enlightening me 👍